content/en/v1.2-docs/configuration/acme/_index.md

```diff
@@ -136,12 +136,16 @@ ACME server
 > Note: In _most_ cases, the MAC key must be encoded in `base64URL`. The 
 > following command will base64-encode a key and convert it to `base64URL`:
 > 
->     $ echo 'my-secret-key' | base64 -w0 | sed -e 's/+/-/g' -e 's/\//_/g' -e 's/=//g'
+> ```console
```

Comment:
Praise: the code block types (there's probably an actual name for it) that you started using for docs are great!